### DISCUSSION TITLE
Thoughts on the project page mockup?

### DISCUSSION THREAD

**Discussion by EnZon3** [EnZon3/Drop#16] [Ideas]
![image](https://github.com/user-attachments/assets/6715e03f-257a-4c5e-b8b8-47058f2f67d4)

**Comment by SalladShooter** [comment]
This looks amazing! Also maybe putting a patterned background behind your whole page (not on the picture), https://uiverse.io/adamgiebl/curvy-earwig-79 or https://uiverse.io/kennyotsu/short-warthog-33. I would also show the creator/owner of each Drop project.

Possible money idea, you could have a way for people to tip each Drop project and you would collect a small percentage 5%-10% (or something) and allow people to make money while you make money.

___

Side note if you need any help with the CSS, just ask and I could create a Pull Request, happy to help where I can.

**Reply by EnZon3** [reply]
Thanks man, that's some valuable feedback. Will update you on the changes

**Reply by EnZon3** [reply]
My idea for monetization would def be tipping on site maybe but also there would be like a pro subscription that would give you some neat community features. Would be 100% balanced tho and would provide no algorithmic advantage

**Reply by SalladShooter** [reply]
Agreed, that would be great, and still keeping a good free tier that contains all of your basic community features that people want (or whatever you plan to do) as it was your goal to do better than Replit.

You could also take Donations here on the GitHub Repo through Sponsors as well.

**Reply by EnZon3** [reply]
But yeah I do need some help with CSS, especially on the landing page, since it doesn't really work well on some screens. That's some feedback I recieved.

**Reply by EnZon3** [reply]
> Agreed, that would be great, and still keeping a good free tier that contains all of your basic community features that people want (or whatever you plan to do) as it was your goal to do better than Replit.
> 
> You could also take Donations here on the GitHub Repo through Sponsors as well.

Yep. I'd have to do some polling around here to determine what to add or what to paywall.

**Reply by SalladShooter** [reply]
Oh, yeah I could help with that!

**Reply by SalladShooter** [reply]
May I ask what the file route is to the main CSS file (I’m going to assume it’s not just Tailwind doing it all), I’m getting kind of lost in the files lol? You can use something like this to achieve what you want -> https://www.w3schools.com/html/html_responsive.asp

**Reply by EnZon3** [reply]
> May I ask what the file route is to the main CSS file (I’m going to assume it’s not just Tailwind doing it all), I’m getting kind of lost in the files lol? You can use something like this to achieve what you want -> https://www.w3schools.com/html/html_responsive.asp

There is no global CSS file, it's all Tailwind lol.

**Reply by SalladShooter** [reply]
> There is no global CSS file, it's all Tailwind lol.

 ok, might need to learn Tailwind first lol

**Reply by EnZon3** [reply]
> > There is no global CSS file, it's all Tailwind lol.
> 
> 
> 
>  ok, might need to learn Tailwind first lol

Shouldn't be hard. Have fun!

**Comment by EnZon3** [comment]
![image](https://github.com/user-attachments/assets/19be9297-0079-4a90-9975-22c84aed2bc2)
I added the profile link to the side of the title

**Reply by SalladShooter** [reply]
Possibly put it right below the title, so the text would have some room to be enlarged?

**Reply by EnZon3** [reply]
Maybe, I'll see how that looks.

**Reply by MilesWK** [reply]
I like @SalladShooter's idea of putting it below to be enlarged.

**Reply by EnZon3** [reply]
![image](https://github.com/user-attachments/assets/1566b39d-9b6e-47ee-b5c8-835a081b2497)
How's this y'all?

**Comment by MilesWK** [comment]
Oh darn I was hoping it was CSS. I have no clue how to program Tailwind.

**Reply by EnZon3** [reply]
Oh, no worries, Tailwind has great documentation. I didn't know tailwind either before starting this project 😆

**Comment by EnZon3** [comment]
By the way, for some stuff, I do use a component library. Here's the link for it: https://www.shadcn-svelte.com/

**Comment by MilesWK** [comment]
If possible, I think it would be great to increase the logo and name size a
little bit, but if that makes it to big compared to the title it looks
great!

**Reply by EnZon3** [reply]
Got it, thanks!
